src/pages/messages/components/NviCandidatePage.tsx

```diff
@@ -52,20 +48,26 @@ export const NviCandidatePage = () => {
     meta: { errorMessage: t('feedback.error.get_registration') },
   });
 
-  const periodStatus = nviCandidate?.periodStatus.status;
+  const nviListQuery = location.state?.nviListQuery;
+  const thisCandidateOffset = location.state?.currentOffset;
+
+  const hasOffset = typeof thisCandidateOffset === 'number';
+  const navigateCandidateSearchOffset = hasOffset ? Math.max(thisCandidateOffset - 1, 0) : null;
+  const isFirstCandidate = hasOffset && thisCandidateOffset === 0;
 
   const navigateCandidateQuery = useQuery({
-    queryKey: ['navigateCandidates', 3, candidateOffset, nviListQuery],
-    queryFn: () => fetchNviCandidates(3, candidateOffset ?? 0, nviListQuery),
+    enabled: hasOffset,
+    queryKey: ['navigateCandidates', 3, navigateCandidateSearchOffset, nviListQuery],
+    queryFn: () => fetchNviCandidates(3, navigateCandidateSearchOffset ?? 0, nviListQuery),
```

Comment:
ikke helt fornøyd med å måtte falle tilbake til 0 her, da det aldri vil være riktig. sikkert bedre å legge det i en ternary som faller tilbake til undefined for hele funksjonen
```suggestion
    queryFn: () => fetchNviCandidates(3, navigateCandidateSearchOffset, nviListQuery),
```